add_to_app/flutter_module/pubspec.yaml

```diff
@@ -30,4 +30,4 @@ flutter:
   module:
     androidX: true
     androidPackage: dev.flutter.example.flutter_module
-    iosBundleIdentifier: dev.flutter.example.flutterModule
```

Comment:
This PR SGTM. How does this work mechanically? The only app identifiers I see us having created in our account that's close are
dev.flutter.VeggieSeasons, dev.flutter.test.autofill and dev.flutter.test.FidelityTextInput. Does Apple somehow reduce those to the first 2 parts of the rdn and claim that ID space?

Reply:
I'm really not sure how it works, I was surprised too.